rocAL/include/readers/video/video_properties.h

```diff
@@ -55,5 +55,5 @@ typedef struct Properties
 void substring_extraction(std::string const &str, const char delim, std::vector<std::string> &out);
 void open_video_context(const char *video_file_path, Properties &props);
 void get_video_properties_from_txt_file(VideoProperties &video_props, const char *file_path, bool file_list_frame_num);
-void find_video_properties(VideoProperties &video_props, const char *source_path, bool file_list_frame_num);
+void find_video_properties(VideoProperties &video_props, const char *source_path, bool file_list_frame_num, std::vector<std::string> filenames = {});
```

Comment:
When user pass file names they need not pass source path ryt? Can we make source_path default argument? 
What does file_list_frame_num argument does?

Reply:
file_list_frame_num is an argument used when a text file is passed as input and if it contains frame number instead of timestamps the user would set this argument.

Reply:
> When user pass file names they need not pass source path ryt? Can we make source_path default argument?

Should I make this as default argument in external API?